package.json

```diff
@@ -27,7 +27,7 @@
   },
   "homepage": "https://github.com/dydxexchange/dydx.js#readme",
   "dependencies": {
-    "@dydxprotocol/protocol": "1.0.14",
+    "@dydxprotocol/protocol": "1.0.16",
```

Comment:
If you update this you need to also update the versions of the docker image in docker-compose and config.yml